kernel/uGraph.mli

```diff
@@ -60,8 +60,9 @@ val merge_constraints : Constraints.t -> t -> t
 
 val check_constraint  : t -> univ_constraint -> bool
 val check_constraints : Constraints.t -> t -> bool
-val check_eq_sort : t -> Sorts.t  -> Sorts.t -> bool
-val check_leq_sort : t -> Sorts.t -> Sorts.t -> bool
+val check_qconstraints : ?qnorm:(Sorts.QVar.t -> Sorts.Quality.t) -> Sorts.QConstraints.t -> bool
+val check_eq_sort : ?qnorm:(Sorts.QVar.t -> Sorts.Quality.t) -> t -> Sorts.t  -> Sorts.t -> bool
+val check_leq_sort : ?qnorm:(Sorts.QVar.t -> Sorts.Quality.t) -> t -> Sorts.t -> Sorts.t -> bool
```

Comment:
Not sure about carrying the API changes this far down, maybe better to call qnorm in conversion.ml